content/en/docs/concepts/sampling/index.md

```diff
@@ -30,18 +30,93 @@ want to sample. You might want to customize your sampling strategy to:
   to see traces with specific user attributes.
 - **Filter out noise**: For example, you may want to filter out health checks.
 
-## What is tail-based sampling?
+## Terminology
 
-Tail-based sampling is where the decision to sample a trace happens _after_ all
-the spans in a request have been completed. This is in contrast to head-based
-sampling, where the decision is made at the _beginning_ of a request when the
-root span begins processing. Tail-based sampling gives you the option to filter
-your traces based on specific criteria, which isn’t an option with head-based
-sampling.
+It's important to use consistent terminology when discussing sampling. A trace
+or span is considered "sampled" or "not sampled":
+
+- **Sampled**: A trace or span is processed and exported. Because it is chosen
+  by the sampler as a representive of the population, it is considered
+  "sampled".
+- **Not sampled**: A trace or span is not processed or exported. Because it is
+  not chosen by the sampler, it is considered "not sampled".
+
+Sometimes, the definitions of these terms get mixed up. You may find someone
+state that they are "sampling out data" or that data not processed or exported
+is considered "sampled". These are incorrect statements.
+
+## Head Sampling
+
+Head sampling is a sampling technique used to make a sampling decision as early
+as possible. A decision to sample or drop a span or trace is not made by
+inspecting the trace as a whole.
+
+For example, the most common form of head sampling is Consistent Probability
```

Comment:
can we link to the spec of Consistent Probability Sampling here?

Reply:
Doneski!